plugin/src/main/scala/com/softwaremill/clippy/AdviceLoader.scala

```diff
@@ -15,7 +15,8 @@ class AdviceLoader(global: Global, url: String, localStoreDir: File)(implicit ec
   private val OneDayMillis = 1000L * 60 * 60 * 24
 
   private val localStore = new File(localStoreDir, "clippy.json.gz")
-  private val projectAdviceFile = new File(".clippy.json")
+  private val projectRoot = sys.props.get("user.dir").getOrElse(".")
```

Comment:
I did run into problems when running in IntelliJ, it defaults to running the compiler in a different directory than the project root unfortunately.  Adding it to resources might be better. I am open to suggestions.